files/messages/expiry.txt

```diff
@@ -0,0 +1 @@
+It's been several months since you took part in the Software Carpentry and Data Carpentry instructor training class, and we are now wrapping up your cohort.  According to our records, you have not yet completed the final checkout exercises - if you have, and our records are inaccurate, please let me know, and if you haven't, but are still intending to do so, please get in touch as quickly as you can so that we can work out a plan.
```

Comment:
Split the second sentence in two:

"If you haven't completed the checkout exercises, but are still intending to do so, please **get in touch as quickly as you can** so that we can work out a plan.